style.css

```diff
@@ -0,0 +1,205 @@
+@import url('https://fonts.googleapis.com/css2?family=Roboto&display=swap');
+@import url('https://fonts.googleapis.com/css2?family=Crete+Round&display=swap');
+@import url('https://fonts.googleapis.com/css2?family=Poppins:wght@200&display=swap');
+
+.toolbar {
+  display: flex;
+  width: 375px;
+  height: 48px;
+  background-color: #e3e3e3;
+  justify-content: space-between;
+  margin: auto auto;
+}
+
+.hambergur-button {
+  display: flex;
+  flex-direction: row;
+  align-items: flex-start;
+  padding: 9px;
+  cursor: pointer;
+}
+
```

Comment:
- [ ] As we are imitate figma design , I noticed that your design and figma design are not really similar , I know that you can improve on this. :wink: 
**Your Design** 
![yourdesign](https://user-images.githubusercontent.com/88894525/192090401-03e145e4-4f71-499e-b743-53ab60b50de3.png)

**Figma Design** 
![template1](https://user-images.githubusercontent.com/88894525/192090449-b7222748-93b2-4ace-9e32-36be3219a42a.png)